### DISCUSSION TITLE
Melon DS on fdroid?

### DISCUSSION THREAD

**Discussion by pokegamer5547** [rafaelvcaetano/melonDS-android#313] [General]
I love melon DS and I'm trying to use it as my primary ds emulator since it's stable now. But I think ut would be great if users can install it from fdroid. Sure we can install from GitHub too but people prefer fdroid to make sure there are no proprietary code or tracking going on.

**Comment by rafaelvcaetano** [comment]
Installing from fdroid does not give you that guarantee either. But don't worry. Fdroid and Play Store releases will come very soon.

**Reply by pokegamer5547** [reply]
Thanks! And yeah you are also right about that.
